gradle.properties~

```diff
@@ -0,0 +1,9 @@
+
+# Use this property to enable/disable Gradle Daemon (long-lived background process)
+# Reference: https://docs.gradle.org/current/userguide/gradle_daemon.html
+org.gradle.daemon=true
+
+# Use this property if you want to use a JDK other than the one specified in your PATH variable.
+# e.g org.gradle.java.home=/Library/Java/JavaVirtualMachines/jdk1.7.0_80.jdk/Contents/Home
+# e.g org.gradle.java.home=C:/Program Files/Java/jdk1.7.0_80
+# org.gradle.java.home=
```

Comment:
Please revert all changes to this file.

Reply:
Do I have to delete this file or is it empty?

Reply:
You'll have to delete it.